### PR TITLE
RichTextLabel: allow 'scroll_to_line' when scroll_active is 'false'

### LINKED ISSUE #36134: scroll_to_line does not work while scroll active is disabled
(3.2.stable.official) For Rich Text Label, If scroll_active is False, scroll_to_line(n) does not scroll to the line number given. Despite the on hover message for Scroll Active in the Inspector saying 'Setting this to false does not block scrolling completely. See scroll_to_line()'

### PR DESCRIPTION
**Pull request by dreamsComeTrue**
Fixes #36134.

That functionally actually worked previously, but was broken due to fixing other issue: #32964

With my PR both of those stories are satisfied:)

### CODE REVIEW

**Review by KoBeWi — APPROVED** [review]
The code looks ok (except style problems). I tested and it doesn't re-introduce #32964. Should be fine to merge after rebasing probably.

File: scene/gui/scroll_bar.cpp [hunk 1]
```diff
@@ -546,6 +546,9 @@ void ScrollBar::_drag_node_exit() {
 
 void ScrollBar::_drag_node_input(const Ref<InputEvent> &p_input) {
 
+	if (!drag_node_enabled)
```
Comment (akien-mga) [inline scene/gui/scroll_bar.cpp:549]: Please add braces around the statement.

File: scene/gui/scroll_bar.cpp [hunk 2]
```diff
@@ -665,6 +672,7 @@ ScrollBar::ScrollBar(Orientation p_orientation) {
 	highlight = HIGHLIGHT_NONE;
 	custom_step = -1;
 	drag_node = NULL;
+	drag_node_enabled = true;
```
Comment (akien-mga) [inline scene/gui/scroll_bar.cpp:675]: This can be initialized directly on declaration in the header (same for other members, feel free to port them).

Comment (dreamsComeTrue) [reply scene/gui/scroll_bar.cpp]: Moved all initializers to respective header file.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**